packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -294,21 +403,18 @@ class _TooltipOverlay extends StatelessWidget {
           ),
           child: FadeTransition(
             opacity: animation,
-            child: Opacity(
-              opacity: 0.9,
-              child: ConstrainedBox(
-                constraints: BoxConstraints(minHeight: height),
-                child: Container(
-                  decoration: BoxDecoration(
-                    color: darkTheme.backgroundColor,
-                    borderRadius: BorderRadius.circular(2.0),
-                  ),
-                  padding: padding,
-                  child: Center(
-                    widthFactor: 1.0,
-                    heightFactor: 1.0,
-                    child: Text(message, style: darkTheme.textTheme.body1),
-                  ),
+            child: ConstrainedBox(
+              constraints: BoxConstraints(minHeight: height),
+              child: Container(
```

Comment:
Would this work with `alignment: Alignment.center` instead of the Center child? Not sure if the Center's width/height factors have an effect on how the text is laid out